Lib/idlelib/editor.py

```diff
@@ -270,6 +280,32 @@ def __init__(self, flist=None, filename=None, key=None, root=None):
         self.askinteger = tkSimpleDialog.askinteger
         self.showerror = tkMessageBox.showerror
 
+
+        #init merged extentions binds - needs to be done after color is set
```

Comment:
Can you explain comment?

Reply:
Code context and some of the other extensions depend on frame.text being packed, which occurs in color_config, called from ResetColorizer. If done before then, idle errors out.